### PR TITLE
Add an error tracer for symbolic evaluation

### PR DESCRIPTION
**Pull request by sorawee**

### CODE REVIEW

File: rosette/lib/profile/record.rkt [hunk 1]
```diff
@@ -113,25 +113,23 @@
...
+  (lambda (loc proc in)
+    (record-enter! loc proc in)
+    (call-with-exception-handler
+     (lambda (e)
+       (record-exit! '())
+       e)
+     (thunk
+      (define ret (call-with-values proc* list))
+      (record-exit! ret)
+      (apply values ret)))))
```
Comment (jamesbornholt) [inline rosette/lib/profile/record.rkt:126]: The macro here was intentional -- at the time we changed to this, the compiler wasn't great at optimizing keyword arguments through thunks, which made some benchmarks very slow. Can you test whether profiling performance regresses with this change? If not, fix the comment here which is now out of date.

File: rosette/lib/trace/raco.rkt [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+(define module-name (make-parameter 'main))
+(define file
+  (command-line
+   #:program (short-program+command-name)
+   ; Tool configuration
+   #:help-labels "" "Tracing settings"
+   #:once-each
+
+   ;; SymPro options
```
Comment (jamesbornholt) [inline rosette/lib/trace/raco.rkt:27]: stale comment

Comment (sorawee) [reply rosette/lib/trace/raco.rkt:27]: That's actually intentional. I meant to share flags that are similar to SymPro as much as possible.

File: rosette/lib/profile/record.rkt (hunk 1 above)
Comment (sorawee) [reply rosette/lib/profile/record.rkt:126]: I will just revert it back.

**Comment by emina** [comment]
Thanks @sorawee!

**Pull request merged**